bcbio/wgbsseq/deduplication.py

```diff
@@ -13,7 +13,8 @@ def dedup_bismark(data):
     """
     config = data["config"]
     input_file = datadict.get_work_bam(data)
-    input_file = bam.sort(input_file, config, order="queryname")
+    # don't sort even by read names
```

Comment:
Perhaps telling why sorting isn't a good idea would be useful.

Reply:
Hi @lbeltrame!

some samples fail bismark when you alter the bam in any way
https://github.com/FelixKrueger/Bismark/issues/360

S